### ISSUE TITLE
WebmMuxing missing segments_muxed

### ISSUE THREAD

**Issue by keithprickett** [bitmovin/bitmovin-api-sdk-python#1]
I'm unable to get the number of segments muxed via the WebmMuxing object.  The Fmp4Muxing object, as a related example, has this attribute.  Is there a reason why this attribute is missing?

As a curiosity, I looked at an example muxing on your dashboard and it actually shows the value.  I was also able to retrieve the value via a REST call directly.

![image](https://user-images.githubusercontent.com/6335288/69468353-a5443480-0d40-11ea-8d86-56029ac64341.png)

Request URL: https://api.bitmovin.com/v1/encoding/encodings/614b18ec-33f9-4f66-a211-24e0d26a4ccf/muxings/webm?limit=100

output json snippet:

```
               {
                    "avgBitrate": 68889,
                    "id": "46197847-2ecc-4d30-a0f6-7a6f85efd240",
                    "initSegmentName": "init.hdr",
                    "maxBitrate": 70410,
                    "minBitrate": 68002,
                    "outputs": [
                        {
                            "outputId": "310d272a-c842-4650-af3b-ef58f847375d",
                            "outputPath": "dev_keith/bitmovin_eo/614b18ec-33f9-4f66-a211-24e0d26a4ccf/vp9/webm/audio/64000/"
                        }
                    ],
                    "segmentLength": 4.0,
                    "segmentNaming": "seg_%number%.chk",
                    "segmentsMuxed": 4,
                    "streamConditionsMode": "DROP_MUXING",
                    "streams": [
                        {
                            "streamId": "0d597ff3-f548-4e00-9ec6-113fcecfcd35"
                        }
                    ]
                },
```

**Comment by wabiloo** [comment]
Found that `segmentsMuxed` was missing from the OpenAPI model.
I fixed the model with https://github.com/bitmovin/bitmovin-open-api/pull/597.

I leave you guys to do the rest now

**Comment by wabiloo** [comment]
@keithprickett - The issue has been resolved with v1.30 released today.
